app/modules/import-export-customization/assets/js/import/components/import-kit-parts-selection.js

```diff
@@ -35,6 +36,26 @@ export default function ImportKitPartsSelection() {
 		}
 	};
 
+	useEffect( () => {
+		if ( data.uploadedData ) {
+			const includes = [];
+
+			if ( data.uploadedData?.manifest?.[ 'site-settings' ] ) {
+				includes.push( 'settings' );
+			}
+
+			if ( data.uploadedData?.manifest?.templates ) {
+				includes.push( 'templates' );
+			}
+
+			if ( data.uploadedData?.manifest?.content ) {
+				includes.push( 'content' );
+			}
+
+			dispatch( { type: 'ADD_INCLUDES', payload: includes } );
+		}
+	}, [ data.uploadedData, dispatch ] );
```

Comment:
Maybe we should add this to the hook?

https://github.com/elementor/elementor/pull/32099/files#diff-1519ac245d634ff089b8d24098eaa6fb150c29caffaed6d85c7bc6b9309fd110R54-R56